docs/docsite/rst/dev_guide/developing_python3.rst

```diff
@@ -295,7 +295,7 @@ new exception-catching syntax which uses the ``as`` keyword:
 Do **not** use the following syntax as it will fail on every version of Python-3:
 
 .. This code block won't highlight because python2 isn't recognized. This is necessary to pass tests under python 3.
-.. code-block:: python2
```

Comment:
@abadger I believe you made this change

Reply:
Hmm... there's a python3 lexer which implies that the plain python lexer is python2.... On my local machine with sphinx-1.5.2 installed, .. code-block:: python appears to work.  I wonder if there's a version mismatch on the builder?  Or if it depends on python2 being installed somewhere and on the python3 builders only python3 is installed?

Reply:
On my local machine, in fact, both code-block python and python3 highlight that line.  Try using python and testing on the ubuntupy3 builder to see if it throws a warning or not.